### PR TITLE
chore: bump Node version to `node20`

### PR DESCRIPTION
**Pull request by luojiyin1987**

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ inputs:
   overwrite:
     description: "Overwrite target container image with slimmed version (only if target is not ID)"
     required: false
-    default: false
+    default: "false"
```
Comment (dwisiswant0) [inline action.yaml:26]: Why is this wrapped in double quotes?

Comment (luojiyin1987) [reply action.yaml:26]: image

A string representing the default value. The default value is used when an input parameter isn't specified in a workflow file.

vscode code get me  error info.

Comment (dwisiswant0) [reply action.yaml:26]: Confirmed.

https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault

**Comment by luojiyin1987** [comment]
https://github.com/slimtoolkit/slim    , I am read the code  to solve [my  problem](https://github.com/kaiyuanshe/OSS-toolbox/pull/20) . "next no find"

**Pull request merged**